lizrd/scripts/generate_gpu_max_batch_size_report.py

```diff
@@ -0,0 +1,90 @@
+import yaml
+import argparse
+import json
+import pandas as pd
+
+from lizrd.support.misc import merge_dicts
+
+
+def main(
+    gradient_checkpointing: bool,
+    group_sizes: [int],
+    models: [str],
+    report_files: [str],
+    output_file,
+):
```

Comment:
I really need a docstring here.
What are the inputs? outputs?
what fields does the yaml config require?

Reply:
outdated